Assets/MRTK/Core/Providers/UnityInput/MouseDeviceManager.cs

```diff
@@ -96,9 +96,9 @@ public float WheelSpeed
         /// <inheritdoc />
         public override void Initialize()
         {
-            base.Initialize();
-
             ReadProfile();
+
+            base.Initialize();
```

Comment:
Along the lines of https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8906/files#r518260139, wondering why this was moved to the end. Most other `base` calls at the end of the methods are called out with

>// Call the base after initialization to ensure any early exits do not
>// artificially declare the service as initialized.

but this one doesn't appear to have a failure case.

Reply:
thanks for pointing that out. it was moved in error.